WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -358,6 +369,40 @@ - (NSString *)addSurveyServerCustomSettingsToURLString:(NSString *)baseURLString
   return baseURLString;
 }
 
+- (NSString *)declineParamsWithEndUserID:(long)endUserID userID:(NSNumber *)userID accountID:(NSNumber *)accountID uniqueLink:(nonnull NSString *)uniqueLink priority:(int)priority {
+  NSString *params = [NSString stringWithFormat:@"origin_url=%@&end_user[id]=%ld&survey[channel]=mobile&survey[unique_link]=%@&priority=%i", _settings.originURL, (long)endUserID, uniqueLink, priority];
+  
+  if (userID != nil) {
+    params = [NSString stringWithFormat:@"%@&user_id=%ld", params, [userID longValue]];
+  }
+  
+  if (accountID != nil) {
+    params = [NSString stringWithFormat:@"%@&account_id=%ld", params, [accountID longValue]];
+  }
+  return params;
+  
+}
+
+- (NSString *)responseParamsWithScore:(NSInteger)score endUserID:(long)endUserID userID:(NSNumber *)userID accountID:(NSNumber *)accountID uniqueLink:(nonnull NSString *)uniqueLink priority:(int)priority text:(nullable NSString *)text {
```

Comment:
This methods share 80% of their code, take it or leave it but I think you could minimize the repetition here. As I stated, take it or leave it...